src/pages/Pool/index.tsx

```diff
@@ -193,6 +195,12 @@ function WrongNetworkCard() {
 export default function Pool() {
   const { account, chainId } = useWeb3React()
   const toggleWalletModal = useToggleWalletModal()
+  const { analytics } = useAnalyticsContext()
+
+  // Segment Page view analytics
+  useEffect(() => {
+    analytics.track('Swap Page viewed')
```

Comment:
Suggestion: make an enum that defines all those strings so that it's centralized in one place.

```Typescript
export enum {
  SWAP = 'Swap Page viewed',
  // ... etc ... other pages
}
```